zones/stardewmodding.wiki

```diff
@@ -0,0 +1,32 @@
+$TTL 300
+$ORIGIN stardewmodding.wiki.
+
+@		SOA ns1.miraheze.org. hostmaster.miraheze.org. (
+		20210416000001	; serial
+		7200		; refresh
+		30M		; retry
+		3D		; expire
+		900		; ncache
+)
+
+; Wildcard services
+@		DYNA	geoip!cp
+
+; Name servers
+@		NS	ns1.miraheze.org.
+@		NS	ns2.miraheze.org.
+
+; CAA (issue: letsencrypt.com, iodef: operations)
+@		TYPE257 \# 22 000569737375656C657473656E63727970742E6F7267
+@		TYPE257 \# 37 0005696F6465666D61696C746F3A6F7065726174696F6E73406D69726168657A652E6F7267
+
+; Mail exchangers
+
+; Servers
+
+; Services
+www		DYNA	geoip!cp
```

Comment:
If a 'www' is not requested there's not really a need to add it to services, though this is minor.

Reply:
> If a 'www' is not requested there's not really a need to add it to services, though this is minor.

@Reception123 I mean, since we've added a wildcard record for `geoip!cp`, the thinking was we could set this up as a wildcard cert and have all the other subdomains redirect to the user's wiki, but yeah, I can remove that.

Reply:
Done.